selftests/checkall

```diff
@@ -5,6 +5,7 @@ run_rc() {
     eval $1
     if [ $? != 0 ]; then
         GR=1
+        [ ! "$SELF_CHECK_CONTINUOUS" ] && exit 1
```

Comment:
From the card I understood it's suppose to fail only between checks and not while executing the tests, but I'm open to suggestions here. I actually welcome using the `failfast=True` when `SELF_CHECK_CONTINOUS` is defined, but I'd prefer not to adjust the `AVOCADO_SELF_CHECK` branch. What are your ideas?